pyndemic/core/api.py

```diff
@@ -0,0 +1,59 @@
+from enum import Enum
+
+
+class CommandTypes(Enum):
```

Comment:
I think CommandTypes enum containing COMMAND element is confusing. Maybe call it RequestTypes?